cpp/test/rank.cc

```diff
@@ -51,3 +51,18 @@ TEST(RankTest, TestRankDescription) {
   ASSERT_FALSE(a.isFlush());
   ASSERT_TRUE(b.isFlush());
 }
+
+TEST(RankTest,TestRankDescription){
```

Comment:
The problem here is `TestRankDescription` has been used and google test blocks us from reusing the same test name.

To fix this, I'd suggest changing the name to `TestRankDescriptionStraight`.